examples/pipelines/fl_cross_silo_factory/submit.py

```diff
@@ -42,9 +42,6 @@
 # to handle yaml config easily
```

Comment:
Should we remove the term "factory" from this submit.py entirely? (and in our docs as well?)

Reply:
done! removed from everywhere